Cakefile

```diff
@@ -135,7 +129,7 @@ task 'build:browser', 'merge the built scripts into a single file for use in a b
       var CoffeeScript = function() {
         function require(path){ return require[path]; }
         #{code}
-        return require['./coffee-script'];
+        return require['./browser'];
```

Comment:
This is for logical consistency. Since `browser.js` is already used as the entry for bower and webpack. It could avoid unnecessary confusion to use it for all browser related situations.